plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeOutputTableHandle.java

```diff
@@ -106,4 +109,10 @@ public Optional<String> getComment()
     {
         return comment;
     }
+
+    @JsonProperty
+    public Optional<Boolean> isChangeDataFeedEnabled()
```

Comment:
As discussed offline, to get deserialized by Jackson binding.

```suggestion
    public Optional<Boolean> getChangeDataFeedEnabled()
```